src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -874,7 +874,7 @@ private static byte[] GetSendBuffer(int bufferSize)
                 sendBuffer[i] = (byte)((int)'a' + i % 23);
             }
 
-            if (bufferSize == DefaultSendBufferSize && s_DefaultSendBuffer == null)
+            if (bufferSize == DefaultSendBufferSize && s_DefaultSendBuffer.Length == 0)
```

Comment:
It would have already returned from above when `bufferSize == DefaultSendBufferSize`.
This changes the behavior of `GetSendBuffer` method, so the functionality that uses `GetSendBuffer` may not work anymore -- possibly the related parameters don't make sense anymore, such as `-BufferSize`.

Reply:
Yes, this part is no longer needed.  `GetSendBuffer()` is still needed when using non-default buffer size.